dotCMS/src/main/webapp/html/portlet/ext/osgi/available_bundles_json.jsp

```diff
@@ -18,7 +19,14 @@
 %>
 
 <%
-    String f = FileUtil.getRealPath( "/WEB-INF/felix/undeployed" );
+    String path;
+    try {
+        path = OSGIUtil.getInstance().getBundleContext().getProperty(OSGIUtil.FELIX_UNDEPLOYED_DIR);
+    } catch (Exception ex) {
+        path = "/WEB-INF/felix/undeployed";
+    }
```

Comment:
Or make the new method "getUndeployedPath" public so this jsp can call it as well